docs/source/developer/testing.md

```diff
@@ -0,0 +1,17 @@
+# Testing
+
+## Customizing app settings on tests
+
+You can use pytest markers to overwrite specific application settings
+on tests, for instance:
+
+```python
+@pytest.mark.app_settings({
```

Comment:
I think maybe a bad rebase here? We don't ahve this in 4.x!

Reply:
Oh, I see. It's backported now! Awesome!

Reply:
:)